_posts/2017-03-28-bettracker-typescript.md

```diff
@@ -48,7 +48,7 @@ Following the logic that TypeScript was just a superset of JavaScript, we were h
 
 On first pass, we were presented with about 1800 errors all looking a bit like the following:
 
-```bash
+```
```

Comment:
I know these aren't actually `bash`, but adding this as syntax highlighter does seem to give sensible highlighting.